File: src/com/vitorblog/mineshop/model/event/ItemDeliveryEvent.java
package com.vitorblog.mineshop.model.event;

import br.com.mineshop.msdk.webservice.endpoints.v1.QueueItem;
import org.bukkit.event.Event;
import org.bukkit.event.HandlerList;

public final class ItemDeliveryEvent extends Event {
    private static final HandlerList handlers = new HandlerList();
    private QueueItem queueItem;

    public ItemDeliveryEvent(QueueItem queueItem) {
        this.queueItem = queueItem;
    }

    public QueueItem getQueueItem() {
        return queueItem;
    }

    public HandlerList getHandlers() {
        return handlers;
    }

    public static HandlerList getHandlerList() {
        return handlers;
    }
}

File: src/com/vitorblog/mineshop/model/event/ItemDeliveryEvent.kt
package com.vitorblog.mineshop.model.event

import br.com.mineshop.msdk.webservice.endpoints.v1.QueueItem
import org.bukkit.event.Event
import org.bukkit.event.HandlerList

class ItemDeliveryEvent(val queueItem:QueueItem) : Event() {

    private val handlers = HandlerList()

    override fun getHandlers(): HandlerList {
        return handlers
    }

    fun getHandlerList(): HandlerList {
        return handlers
    }

}